themes/freemind/source/css/style.css

```
@@ -27,164 +27,208 @@ a {
   text-align: center;
   text-decoration: none;
+  font-weight: 600;
 }
 .rsspart a:hover {
-  color: #39FF14;
-  background: #fafafa;
+  background: #7dd3fc;
+  border-radius: 6px;
 }
 .rsspart a::before {
   font-family: "FontAwesome";
```

Stylelint (17.3.0)

[error] Unexpected quotes around "FontAwesome" (font-family-name-quotes)

(font-family-name-quotes)